robocop/checkers/errors.py

```diff
@@ -217,6 +217,15 @@
         severity=RuleSeverity.ERROR,
         version=">=5.0",
     ),
+    "0415": Rule(
+        rule_id="0415",
+        name="invalid-section-in-resource",
+        msg="Resource file with '{{ section_name }}' section is invalid",
```

Comment:
I think I would go with something similar what is in the docs below, like:
`Resource file can't contain '{{ section_name }}'`. It's so short and easier to understand that more general "Resource file with (section) is invalid", because it does not provide the reason.

What do you think?

Reply:
This error is literally what Robot Framework already reports (and we're catching this as parsing-error). But I like ``Resource file can't contain '{{ section_name }}'`` better so I will use it